src/using-workflow/using-docker-images.md

```diff
@@ -33,13 +33,9 @@ In order to deploy Docker images, they must conform to the following requirement
 Use `deis create` to create an application on the [controller][].
 
     $ mkdir -p /tmp/example-go && cd /tmp/example-go
-    $ deis create
+    $ deis create example-go --no-remote
     Creating application... done, created example-go
 
-!!! note
-    The `deis` client uses the name of the current directory as the
-    default app name.
-
```

Comment:
that's because you're in `/tmp/example-go`, so the name is inferred from the subdirectory. If you were in `tmp/foo` you'd get a 404 NOT FOUND error because the app name `foo` was not found.